wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -294,8 +294,8 @@ pub enum WalletCommand {
     #[clap(name = "staking-list-created-block-ids")]
     ListCreatedBlocksIds,
 
-    /// Get the current staking VRF key for this account
-    #[clap(name = "staking-get-vrf-key")]
+    /// Show the issued staking VRF keys for this account
+    #[clap(name = "staking-show-vrf-key")]
```

Comment:
staking-show-vrf-keys

Reply:
Also, make it possible to show the old, legacy key. Call it "staking-show-legacy-vrf-key", and write on the documentation "Shows the legacy VRF key that uses an abandoned derivation mechanism. This will not be used for new pools and should be avoided".